osbuild/solver/dnf5.py

```diff
@@ -470,6 +470,7 @@ def depsolve(self, arguments):
             "solver": "dnf5",
             "packages": packages,
             "repos": repositories,
+            "modules": {},
```

Comment:
The commit message says

> the osbuild/images implementation always expects the "modules" to be set in the response

which I think is a problem.  Let's keep the depsolvers returning empty modules, that's fine, but let's also make sure the code in images is fine with missing properties in the response.